src/components/Form/FormFormats.js

```diff
@@ -1475,6 +1447,48 @@ export default {
                 const {
                     instructor: { user },
                 } = course;
+
+                const isAtLeastTwoCourseAvailabilities =
+                    course.activeAvailabilityList.length >= 2;
+                const isAtLeastThreeCourseAvailabilities =
+                    course.activeAvailabilityList.length >= 3;
+
+                const courseAvailabilities = {
+                    'dayOfWeek-1': course.activeAvailabilityList[0].dayOfWeek,
+                    'startTime-1': moment(
+                        course.activeAvailabilityList[0].startTime,
+                        'HH:mm'
+                    ),
+                    'endTime-1': moment(
+                        course.activeAvailabilityList[0].endTime,
+                        'HH:mm'
+                    ),
+                    ...(isAtLeastTwoCourseAvailabilities && {
+                        'dayOfWeek-2':
+                            course.activeAvailabilityList[1].dayOfWeek,
+                        'startTime-2': moment(
+                            course.activeAvailabilityList[1].startTime,
+                            'HH:mm'
+                        ),
+                        'endTime-2': moment(
+                            course.activeAvailabilityList[1].endTime,
+                            'HH:mm'
+                        ),
+                    }),
+                    ...(isAtLeastThreeCourseAvailabilities && {
+                        'dayOfWeek-3':
+                            course.activeAvailabilityList[2].dayOfWeek,
+                        'startTime-3': moment(
+                            course.activeAvailabilityList[2].startTime,
+                            'HH:mm'
+                        ),
+                        'endTime-3': moment(
+                            course.activeAvailabilityList[2].endTime,
+                            'HH:mm'
+                        ),
+                    }),
+                };
```

Comment:
Can probably generate this object with a `reduce` on `course.activeAvailabilityList` to avoid repeating yourself.